lib/spack/spack/build_environment.py

```diff
@@ -59,6 +59,11 @@
 SPACK_DEBUG_LOG_DIR    = 'SPACK_DEBUG_LOG_DIR'
 
 
+# Platform-specific library suffix.
+dso_suffix = 'dylib' if sys.platform == 'darwin' else 'so'
```

Comment:
Unfortunately, this does not solve the whole problem.  libfoo version 1.0.2
is called `libfoo.so.1.0.2` on Linux, and `libfoo.1.0.2.dylib` on Mac.
This issue broke multiple sanity-checks with EasyBuild on Mac.

It's probably also worth considering whether the dot should be part of the
dso_suffix variable, or separate.

-- Elizabeth

On Sun, Mar 27, 2016 at 12:58 AM, Denis Davydov <notifications@github.com>
wrote:

> In lib/spack/spack/build_environment.py:
>
> > @@ -59,6 +59,11 @@
> >  SPACK_DEBUG_LOG_DIR    = 'SPACK_DEBUG_LOG_DIR'
> >
> >
> > +# Platform-specific library suffix.
> > +dso_suffix = 'dylib' if sys.platform == 'darwin' else 'so'
>
> it's good to have this one around.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/657/files/7863cdaf7069cc9fae448d9efb0ead2db40f54d6#r57523464>
>